src/main/java/duke/DukeException/DukeException.java

```diff
@@ -0,0 +1,23 @@
+package duke.DukeException;
+
+/**
+ * Handle reply for unknown input given by the user.
+ */
+public class DukeException extends Exception{
+    /**
+     * Create a exception with specified reply message.
+     * @param ErrMsg Reply message that will be displayed.
+     */
+    public DukeException(String ErrMsg) {
+        super(ErrMsg);
+    }
+
+    /**
+     * Return the reply message as a string.
+     * @return String that represent the reply message.
+     */
+    @Override
+    public String toString() {
+        return super.getMessage();
+    }
+}
```

Comment:
Should the JavaDoc start with a verb-s instead of imperative? 

For instance, instead of `Create`, we write `Creates`. Instead of `Return`, we write `Returns`, and so on.

This issue is also present in the `@return` description.

You may view the exact guide [here](https://se-education.org/guides/conventions/java/intermediate.html#comments)